### PR TITLE
[KYUUBI #4618][REST] Admin Resource list operations with sessionHandle filter

### LINKED ISSUE #4618: [Improvement][REST] AdminResource add get Operations with filter
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

`api/v1/admin/operations` add sessionHandler queryparams

### How should we improve?

_No response_

### Are you willing to submit PR?

- [X] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

Close #4618 
`api/v1/admin/operations` add queryParams sessionHandle

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zwangsheng** [comment]
cc @pan3793 @turboFei @lightning-L

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
There is room for performance improvement but is good to get in.

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala [hunk 1]
```diff
@@ -160,7 +160,9 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
       "get the list of all active operations")
```
Comment (Yikf) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala:160]: Please update the API description.

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala [hunk 2]
```diff
@@ -173,6 +175,10 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
       val usersSet = users.split(",").toSet
       operations = operations.filter(operation => usersSet.contains(operation.getSession.user))
     }
+    if (StringUtils.isNotBlank(sessionHandle)) {
```
Comment (Yikf) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala:178]: Agree with @pan3793 , we can refactor this method to improve performance.

Comment (zwangsheng) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala:178]: Sure, i'll make a follow up PR fix those.

**Pull request closed without merging**